README.md

```diff
@@ -261,6 +262,22 @@ Generate token to replace in the project.
 
 ## Deploy application
 
+### Deploy with docker compose
+
+* build create jar: `./gradlew clean bootJar`
```

Comment:
Maybe out of scope of this PR, but you might want to do this using docker too, then the build process will be decoupled from the host environment.

Reply:
I will take care about that in another PR.